corfu_scripts/corfu_layouts.clj

```diff
@@ -81,7 +81,8 @@ Options:
                                              (try
                                                (.get (.bootstrapLayout (get-layout-client) new-layout))
                                                (catch Exception e
-                                               (println server":" (.getMessage e))))
+                                                 (println server":" (.getMessage e))
+                                                 (throw e)))
```

Comment:
This prevents adding a new node to the cluster.
Start a new node on 9000 as single node and 9001 without being bootstrapped.
`./corfu_server -ms 9000`
`./corfu_server -ms -M localhost:9000 9001`

Now if we try to edit 9000's layout to add 9001 we get an AlreadyBootstrappedException for 9000 and it stops there.